Formula/asn.rb

```diff
@@ -0,0 +1,22 @@
+class Asn < Formula
+  desc "Organization lookup and server tool (ASN / IPv4 / IPv6 / Prefix / AS Path)"
+  homepage "https://github.com/nitefood/asn"
+  url "https://raw.githubusercontent.com/nitefood/asn/64e92533f57e9261d1d6588b4714a4183fc3c304/asn"
+  version "0.71.6"
+  sha256 "277765b43a44ccf53df5d60437f7564f003d86d9c4e32e98b9eb0c8b409cf02e"
+  license "MIT"
+
+  livecheck do
+    skip "Cannot reliably check for new releases upstream"
+  end
+
+  bottle :unneeded
+
+  def install
+    bin.install "asn"
+  end
+
+  test do
+    assert_match version.to_s, shell_output(%Q|sed -n 's/^ASN_VERSION="\\(.*\\)"$/\\1/p' #{bin}/asn|)
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

This doesn't even seem like a test of `asn` so much as a test of `sed`.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula